open-craft/edx-platform
# BB-2166 Adds CELERY_CHECK_ROUTING_KEY to cms envs

Addresses issue with CMS worker task failures noted in [open-craft:edx-platform#215](https://github.com/open-craft/edx-platform/pull/215/files#r400609742).

**Testing instructions**

Deployed on my client's in-progress instance (cf [SE-2411](https://tasks.opencraft.com/browse/SE-2411) for testing, and it resolves the issues noted.

Confirm by shelling into that server and running:

```bash
# confirm that all edxapp_worker processes are running.
$ /edx/bin/supervisorctl status

# confirm this setting is available in the CMS config
/edx/bin/edxapp-shell-cms
>>> from django.conf import settings
>>> settings.CELERY_CHECK_ROUTING_KEY
'edx.cms.core.high'
**Reviewer**

- [ ] @alanoe or @kaizoku
